scripts/localbuild.sh

```diff
@@ -134,7 +154,7 @@ source scripts/processsourcefiles.sh
 
 # BUILD MARKDOWN
 # Start HUGO build
-hugo $SERVER --baseURL "" --environment "$BUILDENVIRONMENT"
+hugo $SERVER --baseURL "" --environment "$BUILDENVIRONMENT" $LIVERELOAD --gc
```

Comment:
primary change (3/3) -  also added `--gc` to clean cache each time (ie. using live reload is useful only for site updates) https://gohugo.io/commands/hugo_server/